doc/changelog.rst

```diff
@@ -58,6 +59,7 @@ Admin
 Front
 ~~~~~
 
+- Move compute_bought_with_relations management command under core
 - Add management command to generate bought with relations
```

Comment:
Just remove the "Add management command..." line, since change log should be written from the point of view of the last release.